parquet-column/src/test/java/org/apache/parquet/column/values/bloom/TestBloom.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.parquet.column.values.bloom;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.apache.parquet.column.values.RandomStr;
+import org.apache.parquet.io.api.Binary;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+
+public class TestBloom {
+  @Test
+  public void testIntBloom () throws IOException {
+    Bloom bloom = new Bloom(279, Bloom.HASH.MURMUR3_X64_128, Bloom.ALGORITHM.BLOCK);
+    for(int i = 0; i<10; i++) {
+      bloom.insert(bloom.hash(i));
+    }
+
+    for(int i = 0; i<10; i++) {
+      assertTrue(bloom.find(bloom.hash(i)));
+    }
+  }
+
+  @Test
+  public void testBinaryBloom() throws IOException {
+    Bloom binaryBloom = new Bloom(
+      0,
+      Bloom.HASH.MURMUR3_X64_128,
+      Bloom.ALGORITHM.BLOCK);
+
+    List<String> strings = new ArrayList<>();
+    RandomStr randomStr = new RandomStr();
+    for(int i = 0; i<100000; i++) {
+      String str = randomStr.get(10);
+      strings.add(str);
+      binaryBloom.insert(binaryBloom.hash(Binary.fromString(str)));
+    }
+
+    for(int i = 0; i<strings.size(); i++) {
+      assertTrue(binaryBloom.find(binaryBloom.hash(Binary.fromString(strings.get(i)))));
+    }
+
+    // exist can be true in a very low probability.
```

Comment:
I do not think this is sufficient to test whether the FPP is around 0.01, as specified, or worse.

I would suggest trying 100,000 more random values and checking to see how many are present. You would expect around 1000, right?